frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/reducers.ts

```diff
@@ -251,6 +251,11 @@ export default (state, action: WizardInternalAction) => {
         [dialogID, 'tabs', VMWizardTab.VM_SETTINGS, 'value'],
         fromJS(payload.value),
       );
+    case InternalActionType.SetTemplateValidations:
+      return state.setIn(
+        [dialogID, 'commonData', 'dataIDReferences', 'templateValidations'],
+        payload.value,
```

Comment:
shouldn't it be `fromJS(payload.value),` ?

Reply:
No, because the payload isn't an immutable object but a `TemplateValidations[]` object.